backend/routes/taskRoutes.js

```diff
@@ -10,4 +11,17 @@ const router = express.Router();
 // @acess Private
 router.post("/", [authenticateToken, createTask]);
 
-module.exports = router;
\ No newline at end of file
+// @desc     Fetch all tasks
+// @route    GET /api/tasks/
+// @access   Private
+router.get("/", [authenticateToken, getTasks]);
+
+// @desc     Fetch single task
+// @route    GET /api/tasks/task/:id
+// @acesss   Private
+router.get("/task/:id", [authenticateToken, getTasksById]);
+
+
```

Comment:
Perhaps it would be good to add in the @desc, @route and @access comment for the /day/:day route

Reply:
Fixed: Changes are reflected in recent commits